CHANGELOG.md

```diff
@@ -21,7 +21,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Restrict Typescript checks for component props @kuzhelov ([#1290](https://github.com/stardust-ui/react/pull/1290))
 
 ### Fix
-- Fixed Flex.Item children prop type @mnajdova ([#1320](https://github.com/stardust-ui/react/pull/1320))
+- Fixed `Flex.Item` children prop type @mnajdova ([#1320](https://github.com/stardust-ui/react/pull/1320))
+
+### Features
+- Add `DropdownItem` `selected`, `isFromKeyboard` props @mnajdova ([#1299](https://github.com/stardust-ui/react/pull/1299))
```

Comment:
nit: just to make it more clear, may be rephrased as:

> Add  `selected`, `isFromKeyboard` props to `DropdownItem`

Otherwise it reads like `DropdownItem` is an added prop as well